### PR TITLE
App restart button

### PR DESCRIPTION
**Pull request by frank-laemmer**
New feature to reset an App. Needs review. Which services will be restarted? In which cases will this help?

### CODE REVIEW

File: docs/app.md [hunk 1]
```diff
@@ -264,42 +283,49 @@ A task that is completed has a check-mark `✔`. Tasks that are still todo will
...
 
+
 ### The end of a trial
 
 When the trial is finally over, the trial App will be deleted. That might sounds cruel, but please consider that: We will not keep your data hostage until you pay us. 
 
 Good news is: we will inform you by e-mail before it happens and you can start a new trial right away, after your current trial ended. In many cases you can easily re-deploy your code base to the new App: When deploying with [Git](git-deployment), you just change the remote and push again. When using our CMS/framework presets with predefined [ENV vars](env-vars) database access will work without code modification.
 
 
+## Resetting an App
```
Comment (einenlum) [inline docs/app.md]: Restarting an App

File: docs/app.md [hunk 2]
```diff
@@ -264,42 +283,49 @@ A task that is completed has a check-mark `✔`. Tasks that are still todo will
...
+
+To reset an App, visit the App in the Dashboard, find and hit the "Reset App" button. This will restart the Apache service and the PHP-FPM processes - nothing more. There will be a short downtime of the App. This can help in certain edge cases, such as persistent 408 errors or hanging PHP processes with 503 errors. Use carefully, it should not be used on a regular basis. Please <a href="javascript:Intercom('showNewMessage')">contact us</a>. if you have ongoing trouble with your App.
```
Comment (einenlum) [inline docs/app.md]: To restart an App

Comment (einenlum) [inline docs/app.md]: Hit the "Restart App" button

Comment (einenlum) [inline docs/app.md]: Should be a comma (,) instead (.) after "contact us".

Comment (einenlum) [inline docs/app.md]: Even if it's implicit with "Nothing more", I would still add explicitely that data is safe.

Comment (frank-laemmer) [reply docs/app.md]: Good catch.

Comment (frank-laemmer) [reply docs/app.md]: Agree.

Comment (frank-laemmer) [reply docs/app.md]: Sure!

**Review by einenlum — APPROVED** [review]
Nice! Thx.

**Review by ostark — COMMENTED** [review]
Not related to "App restart button" at all - but okay.

**Pull request merged**